recipes/dcap/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "dcap" %}
+{% set version = "2.47.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dCache/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 050a8d20c241abf358d5d72586f9abc43940e61d9ec9480040ac7da52ec804ac
+  patches:
+    - 0001-PATCH-Missing-declaration.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - automake
+    - make
+    - pkg-config
+    - libtool
+    - sed
+  host:
+    - gct
+    - krb5
+    - openssl 1.0.2r
```

Comment:
I'm using openssl 1.0 as I'm building something which needs it so I'll migrate over to 1.1 in the feedstock.

I've checked and `run_export` is used correctly in `openssl=1.0.2r`.